ironfish-cli/src/commands/wallet/multisig/create-signature-share.ts

```diff
@@ -43,6 +47,13 @@ export class CreateSignatureShareCommand extends IronfishCommand {
       signingPackage = await longPrompt('Enter the signing package: ')
     }
 
+    if (!flags.confirm) {
+      const confirmed = await CliUx.ux.confirm('Confirm new signature share creation (Y/N)')
```

Comment:
will we come back later to add some display of what's in the transaction?

Reply:
Yeah, I think that requires a lot of changes. I would rather get some sort of confirmation in right now and revisit lalter.